### ISSUE TITLE
killab33z packages

### ISSUE THREAD

**Issue by noraj** [BlackArch/blackarch#4353]
https://codeberg.org/killab33z/blackarch-pkgs/

- [ ] adpeas
- [ ] breads
- [ ] certi
- [ ] extractbitlockerkeys
- [ ] finduncommonshares
- [ ] gpowned
- [ ] ldapconsole
- [ ] ldapmonitor
- [ ] ldapwordlistharvester
- [ ] mssqlrelay
- [ ] netscout
- [ ] pre2k
- [ ] python-aiowmi
- [ ] rdwatool
- [ ] sccmhunter
- [ ] targetedkerberoast

**Comment by D3vil0p3r** [comment]
Tell me if you need some help here. Meanwhile I push for your current license requests (4368, 4369) by other means.

**Comment by D3vil0p3r** [comment]
For https://github.com/BlackArch/blackarch/pull/4368 , the [LICENSE](https://github.com/p0dalirius/ExtractBitlockerKeys/blob/main/LICENSE) file has been created. I think the PR can be updated and merged.

**Comment by noraj** [comment]
> For [#4368](https://github.com/BlackArch/blackarch/pull/4368) , the [LICENSE](https://github.com/p0dalirius/ExtractBitlockerKeys/blob/main/LICENSE) file has been created. I think the PR can be updated and merged.

It was updated 1 month ago already. I updated #4369 and #4410 today.

**Comment by D3vil0p3r** [comment]
Ja. I have an open channel with him by email. In addition to https://github.com/BlackArch/blackarch/pull/4416 and https://github.com/BlackArch/blackarch/pull/4408 do you need additional need from him upstream?

EDIT: he just added also the LICENSE file to the last mentioned repositories. I guess we should be ok now.

**Comment by D3vil0p3r** [comment]
All PR have been merged. I guess this issue can be closed.
